slices/xz-utils.yaml

```diff
@@ -0,0 +1,26 @@
+package: xz-utils
+
+essential:
+  - xz-utils_copyright
+
+slices:
```

Comment:
It looks like xzgrep and a few of the other utils in `/usr/bin/` maybe shell scripts. Could we move the scripts to a util folder and add the dependencies there?

```
slices:
  bins:
    essential:
      - libc6_libs
      - liblzma5_libs
    contents:
      /usr/bin/lzmainfo:
      /usr/bin/unxz:
      /usr/bin/xz:
slices:
  bins:
    essential:
      - xz-utils_bins
      - dash_bins
      - grep_bins
      ... possibly more dependencies
    contents:
      /usr/bin/xzcat:
      /usr/bin/xzcmp:
      /usr/bin/xzdiff:
      /usr/bin/xzegrep:
      /usr/bin/xzfgrep:
      /usr/bin/xzgrep:
      /usr/bin/xzless:
      /usr/bin/xzmore:
```

Reply:
You mean create a new slice called utils?

```yaml
slices:
...
  utils:
    essential:
      - xz-utils_bins
      - dash_bins
      - grep_bins
```

I just think it's gonna be ugly to call `xz-utils_utils`.

Reply:
I couldn't make it pass the test I've made, but sent the commit anyway so you can help me track what's missing.


I still get this error:

```
chroot: failed to run command ‘xzgrep’: No such file or directory
```

Reply:
I made a rootfs with your slices and busybox so I can take a closer look at the environment, and was able to recreate your error. It looks like xzgrep is expecting /bin/sh to exist. I symlinked /bin to /usr/bin and /usr/bin/sh to /usr/bin/busybox and xzgrep seems to execute now.

I think I saw another slice with a similar hurdle, let me see what the convention is for fixing this.

Reply:
Sorry for the delay. The team mentioned we need to add base-files_bin (or base-files_base) to link /bin to /usr/bin.  I gave it a shot and there were still a few broken lines in the test task. I think some of the process substitution might be breaking under chroot. Tomorrow I'll see if I can come up with an alternate test.

Reply:
Here is a draft PR with some changes I made to get it to run locally. Lets see if it passes spread tests. Feel free to cherry-pick the changes if you like them.

https://github.com/canonical/chisel-releases/pull/338